Makefile.include

```diff
@@ -111,6 +111,14 @@ endif
 
 QQ=
 
+# Set this to 1 to enable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
```

Comment:
You could also add that it enables the `DEBUG_PRINT` macro. https://github.com/RIOT-OS/RIOT/blob/2017.10-branch/core/include/debug.h#L46